elements/pfe-progress-indicator/README.md

```diff
@@ -0,0 +1,44 @@
+# PFElements Progress Indicator Element
+
+`pfe-progress-indicator` is a call-to-action (CTA) element, that stands out from regular hypertext links, and is used for linking users to webpages.
+
+## Usage
+```html
+<pfe-progress-indicator pfe-indeterminate>
+    <h1>My fallback loading message</h1>
+</pfe-progress-indicator>
+```
+
+## Slots
+
+Progress Indicator has one slot only. It is named `progress-message`.
```

Comment:
This might be out of date too - I believe this there is only one unnamed slot, correct?

Reply:
This still indicates that the component has 1 named slot.  Is that accurate?